gateway-api/Cargo.toml

```
@@ -17,6 +17,8 @@ schemars = "0.8.21"
 serde = { version = "1.0.203", features = ["derive"] }
 serde_json = "1.0.118"
 serde_yaml = "0.9.34"
+regex = "1.10.6"
```

```suggestion
regex = { package = "regex-lite", version = "0.1.6" }
```

Just for the sake of compile times and binary sizes 😅